### PR TITLE
Adding local file system as source

### PR DESCRIPTION
**Pull request by pandeyganesh-dc**
Added local file system as a source, which reads data to stdout from storage object.

### CODE REVIEW

File: verified_sources/local_file_system/catalog.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        description='The path of the file to be read.',
+        title='File Path'
+    )
+
+    obj_file_path: Optional[str] = Field(None,
+        json_schema_extra={'ui-opts': {'hidden': True}}
+    )
+
+class LocalFileSystemCatalog(DatCatalog):
+    document_streams: List[Union[TxtDatStream]]
```
Comment (dc-rahul) [inline verified_sources/local_file_system/catalog.py:56]: document_streams should contain the CsvDatStream too

File: verified_sources/local_file_system/specs.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    type: string
+  module_name:
+    description: "Name of the python module for this connector"
+    type: string
+  connection_specification:
+    description: ConnectorDefinition specific blob. Must be a valid JSON string.
+    type: object
+    additionalProperties: true
+    properties:
+      obj_file_path:
```
Comment (dc-rahul) [inline verified_sources/local_file_system/specs.yml:24]: Now that you have moved the usr_file_path and obj_file_path to catalog, so they should be removed from here too

File: verified_sources/local_file_system/catalog.py (hunk 1 above)
Comment (pandeyganesh-dc) [reply verified_sources/local_file_system/catalog.py]: Thanks, I have updated the document_streams

File: verified_sources/local_file_system/catalog.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+type: object
+properties:
+  document_streams:
+    type: array
+    items:
```
[the remark this answers is not in the document]
Comment (pandeyganesh-dc) [reply verified_sources/local_file_system/catalog.yml:5]: catalog.yml also updated with corresponding catalog.py file

File: verified_sources/local_file_system/specs.yml (hunk 2 above)
Comment (pandeyganesh-dc) [reply verified_sources/local_file_system/specs.yml]: removed from the specs.yml

**Review by dc-rahul — APPROVED** [review]

**Pull request merged**